test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -310,3 +311,125 @@ TEST_F(SynchronizerTest, RetrieveBlockTwoFailures) {
 
   ASSERT_TRUE(wrapper.validate());
 }
+
+/**
+ * @given synchronizer receives a commit with the block peer voted for
+ * @when synchronizer processes the commit
```

Comment:
```suggestion
 * @when synchronizer processes the commit @and commit prepared is successful
```

Reply:
Since we test that commitPrepared is called in this test, it is better to put it in a then clause, since it is the effect we are testing